README.md

```diff
@@ -25,40 +27,105 @@ sh ./download_data.sh
+
+You can run `python 1_sequence_prediction.py -h` for the full documentation of inputs.
+
+2. `1_variant_effect_prediction.py` (and corresponding bash script, `1_variant_effect_prediction.sh`): Accepts a VCF file as input and makes variant effect predictions.
+
+Example usage:
 ```
-sh run_pipeline.sh test.vcf hg19 test-output --cuda
+sh 1_variant_effect_prediction.sh <vcf> <hg> <output-dir> [--cuda]
 ```
 
 Arguments:
-- `<example-vcf>`: Input VCF file
-- `<hg-version>`: Reference FASTA. By default the framework accepts `hg19` or `hg38` coordinates.
+- `<vcf>`: VCF file
+- `<hg>`: Either hg19 or hg38
 - `<output-dir>`: Path to output directory (will be created if does not exist)
 - `--cuda`: Optional, use this flag if running on a CUDA-enabled GPU.
 
-We provide `test.vcf` (hg19 coordinates) so you can try running this command once you have installed all the requirements. Additionally, `run_pipeline.gpu_node.sh` is an example SLURM script with the same expected input arguments if you need to submit your job to a compute cluster. 
+You can run `python 1_variant_effect_prediction.py -h` for the full documentation of inputs.
+
+These scripts will output the chromatin profile predictions as HDF5 files to a subdirectory `chromatin-profiles-hdf5` in your specified output directory. 
+
+See `example_slurm_scripts/1_example_seqpred.slurm_gpu.sh` and `example_slurm_scripts/1_example_vep.slurm_gpu.sh` for sample scripts for running chromatin profile prediction on SLURM.
 
-**Additional note**: we have added the capability of predicting variant effects from a pair of sequences in the `vep_cli_seq.py` script in the [`vep_seq`](https://github.com/FunctionLab/sei-framework/tree/vep_seq) development branch of this repo.
 
-### Outputs
+### Sequence class prediction
 
-The following files and directories will be outputted:
--  `chromatin-profiles-hdf5`: directory
--  `chromatin_profiles_diffs.tsv`: chromatin profile prediction TSV file (**Note:** output file will be compressed if input has >10000 variants)
--  `sequence_classes_scores.tsv`: sequence class prediction TSV file 
+Sequence class scores can be obtained from Sei chromatin profile predictions. There are 2 types of scores that can be computed:
 
-The two `*.tsv` files are the final formatted outputs, while the `chromatin-profiles-hdf5` directory contains the intermediate HDF5 and row label files outputted from Selene from running the Sei deep learning model. 
+- Raw sequence class scores: For sequences only. **Note** our manuscript uses the Louvain community clustering, whole-genome sequence class annotation of the human genome whenever we apply sequence classes to reference genome sequences, and we encourage the use of these annotations over the raw sequence class scores when possible. Sequence class annotations for hg38 and hg19 (lifted over from hg38) are available for download from [this Zenodo record](10.5281/zenodo.7113989). 
```

Comment:
maybe add more explanation to the score? I write an example here  "Raw sequence class scores are projection scores of chromatin profile predictions projected on the unit-length vectors representing each sequence class. This is an intermediate score originally developed for variant score prediction and is made available for use for developing downstream analysis or applications such as using them as a sequence representation. "